tutorials/publishing-tutorial/builderdao.lock.json

```diff
@@ -0,0 +1,33 @@
+{
+  "proposalId": 168,
+  "slug": "publishing-tutorial",
+  "creator": "",
+  "authors": [
+    {
+      "name": "The Builder Dao",
+      "avatarUrl": "https://github.com/TheBuilderDAO.png",
+      "url": "https://builderdao.io",
+      "nickname": "TheBuilderDAO"
+    }
+  ],
+  "reviewers": {
```

Comment:
All looks good but I found that `reviewer1` & `reviewer2` both are the same people as per this file wherein actually both are different. Did I miss something here @XnagoR @dgamboa?